src/collections/_documentation/api/organizations/get-organization-users.md

```diff
@@ -1,29 +0,0 @@
----
```

Comment:
Where did this go?

Reply:
Ah. This was added manually: https://github.com/getsentry/sentry-docs/pull/1336.

Going to have to create a scenario to get this back.

Reply:
Dun dun dun doneeee https://github.com/getsentry/sentry/pull/16911/commits/43f265901e895d878a4c8329ae047d8db663d56c